packages/flutter/lib/src/rendering/sliver_persistent_header.dart

```diff
@@ -367,10 +367,13 @@ abstract class RenderSliverScrollingPersistentHeader extends RenderSliverPersist
 
```

Comment:
Looks like the whitespace changes here are leftover? Can you remove them?

Reply:
Done!